test/jdk/javax/swing/JRadioButton/bug4380543.java

```diff
@@ -125,8 +122,8 @@ public void initComponents() throws InterruptedException {
 
         getContentPane().add(buttonsPanel);
 
-        for (Map.Entry mapElement : lookAndFeelMaps.entrySet()) {
-            String btnName = mapElement.getKey().toString();
+        for (Map.Entry<String, String> mapElement : lookAndFeelMap.entrySet()) {
+            String btnName = mapElement.getKey();
```

Comment:
This can be further simplified: you don't use the value, you use only the key, so the for-loop could iterate over `lookAndFeelMap.keySet()`.

In fact, you can avoid the map altogether if you use the class name as the action name:

```java
UIManager.LookAndFeelInfo[] lookAndFeel = UIManager.getInstalledLookAndFeels();
for (UIManager.LookAndFeelInfo look : lookAndFeel) {
    JButton btn = new JButton(look.getName());
    btn.setActionCommand(look.getClassName());
    btn.addActionListener(this);
    p.add(btn);
}
```

In this case, initMap and lookAndFeelMap should be removed.

The listener should be updated:

```java
public void actionPerformed(ActionEvent e) {
    setLookAndFeel(e.getActionCommand());
    SwingUtilities.updateComponentTreeUI(this);
}
```